<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingDialog</class>
 <widget class="QDialog" name="BookingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neue Buchung anlegen</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Preis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="preisInput"/>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Travel ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="travelIDInput"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Von&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>bis</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Typ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="dateEdit1"/>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="dateEdit2"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Flugbuchung</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hotelbuchung</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mietwagenreservierung</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_var1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_var2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_var3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_1"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="smokeInput">
     <property name="text">
      <string>Raucher</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QComboBox" name="seatPrefInput">
     <item>
      <property name="text">
       <string>Fenster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Egal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_var4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="insuranceInput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
